src/cbmc/cbmc_parse_options.h

```diff
@@ -56,6 +57,7 @@ class optionst;
   "(error-label):(verbosity):(no-library)" \
   "(nondet-static)" \
   "(version)" \
+  "(paths)(goto-priority):" \
```

Comment:
Since these are duplicated between cbmc and jbmc and probably other driver programs will want them, suggest moving the arg specs and help text to an appropriate header file and including them here, as is already done for `OPT_GOTO_CHECK` for example.